handlers/deposit_file.go

```diff
@@ -70,12 +70,16 @@ func (d *depositFileEntry) createFileResource(resourceID string, filename string
 
 func (d *depositFileEntry) buildPersistableResource(resourceID string, filename string) *persistence.Resource {
 	resource := &persistence.Resource{ID: resourceID}
-	// TODO: Where should Access come from/default to?
+	// TODO: What should be the default access for a file?
 	resource.Access = "private"
 	resource.AtContext = atContext
 	resource.AtType = fileType
 	resource.Label = filename
 	resource.Preserve = false
 	resource.Publish = false
+	// This will help future proof files in case at some point we decide that the
+	// resource identifier is not a suitable S3 key
+	resource.ContentKey = resourceID
+	resource.ContentBucket = d.rt.Config().S3BucketName
```

Comment:
I don't think the changing uploading behavior or how we persist a resource is appropriate in this commit, nor does it reflect an agreed to change in the shape of a resource.

Reply:
Prior to this commit we have no defined way to store where we persist the resource.  We need to store this data somewhere. Can you tell me what I ought to do?